<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fps_dialog</class>
 <widget class="QDialog" name="fps_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set FPS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/rpcsx.png</normaloff>:/icons/rpcsx.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QLabel" name="fpsName">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string extracomment="0 = unlocked framerate"/>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="text">
       <string>FPS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="fpsSlider">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>31</width>
        <height>181</height>
       </rect>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="pageStep">
       <number>30</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>269</x>
        <y>331</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
     <widget class="QLabel" name="fpsCount">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>unlocked</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="hudDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MangoHud display</string>
      </property>
     </widget>
     <widget class="QLabel" name="overlay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>301</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/icons/overlay.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="settings">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MangoHud Settings</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>fps_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
